cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -1124,6 +1124,127 @@ var (
 			return deleteEndDevice(ctx, devID, skipClusterJS)
 		},
 	}
+	endDevicesBatchDeleteCommand = &cobra.Command{
+		Use:   "batch-delete [application-id] [device-ids]",
```

Comment:
Can we mark this as experimental and come back later to decide if we really need an additional command ? In theory there is nothing stopping us from using the extra identifiers in the existing `delete` for the other devices.